src/app/components/shared/timezone/timezone.component.ts

```diff
@@ -0,0 +1,58 @@
+import { Component, Input, OnChanges } from "@angular/core";
+import { DateTimeTimezone } from "@interfaces/apiInterfaces";
+import {
+  AbstractModel,
+  isUnresolvedModel,
+  UnresolvedModel,
+} from "@models/AbstractModel";
+import { Site } from "@models/Site";
+
+@Component({
+  selector: "baw-timezone",
```

Comment:
I think the name is inappropriate. We are showing a date here. A date is either local or not (has a timezone) but we're still showing the date.

`<baw-date>` would be my preference. I think `<baw-time>`  is too narrow and `<baw-datetime>` is unnecessarilly verbose.